docs/How-to-Create-and-Maintain-a-Tap.md

```diff
@@ -10,23 +10,57 @@ last_review_date: "1970-01-01"
 
 A tap is usually a Git repository available online, but you can use anything as long as it’s a protocol that Git understands, or even just a directory with files in it. If hosted on GitHub, we recommend that the repository’s name start with `homebrew-` so the short `brew tap` command can be used. See the [`brew` manual page](Manpage.md) for more information on repository naming.
 
-The `brew tap-new` command can be used to create a new tap along with some template files.
+The `brew tap-new` command can be used to create a new tap along with some template files:
+
+```console
+$ brew tap-new $YOUR_GITHUB_USERNAME/homebrew-tap
+Initialized empty Git repository in /opt/homebrew/Library/Taps/$YOUR_GITHUB_USERNAME/homebrew-tap/.git/
+...
+==> Created $YOUR_GITHUB_USERNAME/tap
+/opt/homebrew/Library/Taps/$YOUR_GITHUB_USERNAME/homebrew-tap
```

Comment:
What's a user to make of this message:

```
When a pull request making changes to a formula (or formulae) becomes green
(all checks passed), then you can publish the built bottles.
To do so, label your PR as `pr-pull` and the workflow will be triggered.
```

Is that true of any repo that is a tap? Or just homebrew's built-in one?

Reply:
As I understand it, this should apply to any tap that's pushed to GitHub with the auto-created files in `.github/workflows` left in place. See this post for how that plays out: https://brew.sh/2020/11/18/homebrew-tap-with-bottles-uploaded-to-github-releases/